basiclti/basiclti-impl/src/bundle/ltiservice.properties

```diff
@@ -69,14 +69,15 @@ bl_allowoutcomes=Allow External Tool to return grades
 bl_allowlineitems=Allow External Tool to create grade columns
 bl_allowroster=Provide Roster to External Tool
 bl_allowsettings=Allow External Tool to store setting data
-bl_allowcontentitem=Allow External Tool to configure itself (the tool must support the IMS Content-Item or Deep-Link message)
-pl_header=Indicate the following types of Content Item/Deep Link Selection launches this tool can handle.  Not all tools can handle types of launches.  If you enable a tool which is not capable of responding to a particular request, it will likely fail when you try to use it.
-bl_pl_launch=Allow the tool to be launched as a link (this is typically true for most tools)
-bl_pl_linkselection=Allow external tool to configure itself (the tool must support the IMS Content-Item or Deep-Link message)
+pl_header=Tools can generally accept direct LTI Launches or a Content Item / Deep Link Selection launches.  It is not common, but some tools can handle both types of launch at one endpoint.
+bl_allowcontentitem=The tool URL can receive a Content-Item / Deep Link launch
+bl_pl_launch=The tool URL can receive an LTI launch
+bl_pl_linkselection=The tool can receive a ContentItem or Deep Link launch
```

Comment:
ContentItem or Content-Item?

Reply:
Right - I thought I saw all the inconsistent bits - I will fix and resubmit  thanks.

Reply:
It is weird.  If you are using it as an adjective, you use the hyphenated form like "Content-Item Message" - but as a noun, it seems not to be hyphenated.  We have five Content Items in this lesson.

Reply:
but hasn't ContentItem because a proper noun or a brand?